herd/event.ml

```diff
@@ -1839,7 +1840,7 @@ module Make  (C:Config) (AI:Arch_herd.S) (Act:Action.S with module A = AI) :
              [rloc.intra_causality_control;
               rmem.intra_causality_control;rreg.intra_causality_control;
               wmem.intra_causality_control;wreg.intra_causality_control;];
-           if memtag || (physical && is_branching) then
+           if memtag || pac || (physical && is_branching) then
```

Comment:
This may be not for this PR, but it could be good to have tests sensitive to the changes to the logic for SWP and CAS here.

(I conjecture that the task where these changes will show up is checking if dependencies and pick dependencies via PACD* and AUTD* are created correctly.)

Reply:
Upon a consideration of the graph for SWP and CAS, I believe there is no todo item for this PR.